airbyte-integrations/connectors/destination-oracle/src/test-integration/java/io/airbyte/integrations/destination/oracle/OracleIntegrationTest.java

```diff
@@ -165,7 +165,6 @@ protected void tearDown(TestDestinationEnv testEnv) {
 
     final Database database = getDatabase();
     var tables = getAllTables(database);
-    tables.removeAll(allTables);
```

Comment:
This step was strange. We get all tables and remove all of them. The variable `tables` was empty at the end. The function `getAllTables(database)` get all tables where the user is the owner.